src/components/Input/TimeInput.js

```diff
@@ -187,6 +187,10 @@ export default class TimeInput extends React.Component {
 
     const [hasTypedTens, hasTypedMin] = userInputProgress(inputCandidate, time);
 
+    if (input === '' && unavailableTimes?.has(label)) {
```

Comment:
I'm assuming `unavailabeTimes` is a Set based on the tests and the use of `has()`.

I feel like an array would be a better option here. Also, it would help to include it in `propTypes` so there's no ambiguity.

Reply:
Updated.